packages/cadl-ranch/src/actions/validate-mock-apis.ts

```diff
@@ -3,6 +3,7 @@ import pc from "picocolors";
 import { findScenarioCadlFiles, loadScenarioMockApiFiles } from "../scenarios-resolver.js";
 import { importTypeSpec, importCadlRanchExpect } from "../cadl-utils/import-cadl.js";
 import { createDiagnosticReporter } from "../utils/diagnostic-reporter.js";
+import { sep } from "path";
```

Comment:
We should not use any of the native path utils, without normalizing it is better to make sure everything is using `/` otherwise when comparing path it can always result in different result across platforms. I am wondering if those issue you are having are the same as this #382

Reply:
I'll make a separate PR fixing this, so you can revert those changes

Reply:
PR fixing it https://github.com/Azure/cadl-ranch/pull/383

Reply:
Yeah, I think so, I will revert those changes after #383 get merged.

Reply:
updated, should have resolved this comment.